test/init.bats

```diff
@@ -11,6 +11,3 @@ load init
     skip "No test implementation"
 }
 
-@test "load should source extension specific files if possible" {
```

Comment:
Did you have any specific reason to remove this placeholder?
It would be valuable to have this test implemented at some point, that's why this placeholder is there.

Reply:
Yep, actually BATS was reporting an error about this one, because it's duplicate L10 VS L14. I discovered this while running latest version of https://github.com/bats-core/bats-core instead of the old https://github.com/sstephenson/bats that was requested by the Makefile. So I newer versions of BATS does not allow duplicate tests while the old version was not complaining about it. 

TBH, I didn't took the time to run the test with the old BATS version, I upgraded straight to the latest version and I fixed the tests that was not working.